app/src/js/components/home.js

```diff
@@ -23,7 +23,6 @@ import {
   seconds
 } from '../utils/format';
 import List from './Table/Table';
-import GranulesProgress from './Granules/progress';
```

Comment:
This component is almost deprecated. It looks like it's still used on the PDRs page but I don't actually see it rendering. Could it be removed there (/components/Pdr/pdr.js)? If so we can delete the whole thing.

Reply:
Looks like that component is on the PDR page `/pdrs/pdr/MOD09GQ_1granule_v3.PDR`, so I think we should keep it for now. (Unless those need to be updated with card interactions, not sure)
<img width="665" alt="Screen Shot 2020-12-03 at 2 05 44 PM" src="https://user-images.githubusercontent.com/16179480/101075880-99549a80-3570-11eb-98bb-86bd5b89613a.png">

Reply:
Oh it's on the individual PDR page, got it. Hmm yeah, I would think we want these to be the individual cards as well. If you click on one it should bring you to the appropriate granules page but in this case also filtered somehow? I was thinking it would just be filtered by the collection but it looks like that's not quite right.

@jjmccoy what do you think? Danielle's route above has valid data: `/pdrs/pdr/MOD09GQ_1granule_v3.PDR`

Might need to be a separate ticket to swap these out and remove the `Progress` component.

Reply:
@npauzenga Oh okay. Hmm. I didn't realize those cards were associated with the PDR route. Yeah, that change might be out of scope for this ticket. This was supposed to be a quick UI interaction refactor.

Reply:
Ok cool. It's not terribly complicated but it's a little different because we need to filter the Running/Completed/Failed results. If we can do that in a separate ticket that would be great.